core/pom.xml

```diff
@@ -68,7 +68,7 @@
     <!-- Placeholder API Repo -->
     <repository>
       <id>placeholderapi</id>
-      <url>https://repo.extendedclip.com/content/repositories/placeholderapi/</url>
+      <url>https://repo.extendedclip.com/releases/</url>
```

Comment:
I didn't do this when I fixed it on my end, but I'm assuming this still works

Reply:
According to the PAPI wiki, this is the repository they tell you to use more. Not sure why it changed

https://wiki.placeholderapi.com/developers/using-placeholderapi/